src/components/user-group/uc/classes.uc.js

```diff
@@ -0,0 +1,19 @@
+const { classesRepo } = require('../repo');
+
+const getClassesForStudent = async (userId) => {
+	return classesRepo.findClassesByStudent(userId);
+};
+
+const getClassesForTeacher = async (userId) => {
+	return classesRepo.findClassesByTeacher(userId);
+};
+
+const removeUserFromClasses = async (userId, classIds) => {
+	return classesRepo.removeUserFromClasses(userId, classIds);
+};
+
+module.exports = {
+	getClassesForStudent,
```

Comment:
getClassesForStudent 
getClassesForTeacher
Are not uc that are needed for this task and do not include permissions at the moment. I propose that we do not populate it at the moment, until we have permissions for it.